examples/example_differential_evo_regression.py

```diff
@@ -18,8 +18,16 @@
   programming. In European Conference on Genetic Programming
   (pp. 35-51). Springer, Cham.
 
+Usage: 
+  example_differential_evo_regression.py [--max_generations=<N>]
```

Comment:
i would prefer command line argument to be dash-separated like this `--max-generations`

Reply:
done.